modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -431,7 +431,7 @@ public String toEnumValue(String value, String datatype) {
         if ("number".equals(datatype)) {
             return value;
         } else {
-            return "\'" + escapeText(value) + "\'";
+            return "\"" + escapeText(value) + "\"";
```

Comment:
I wouldn't change it to double quotes. instead, https://github.com/swagger-api/swagger-codegen/issues/7369 should be fixed

Reply:
Ok I will rollback this part. I was following the advice of my pro typescript teammate here but I guess it doesn't matter :)
Will provide a new version sometime this morning.
Thanks!